Reneiski_Homework1/Reneiski_Homework1/Program.cs

```diff
@@ -1,28 +1,279 @@
+            static void PrintWordOfLengthLinq(List<string> strings)
+            {
+                Console.WriteLine("Please type the length");
+
+                int lengthFromUser = int.Parse(Console.ReadLine());
+                var lengthOfString = from l in strings where l.Length == lengthFromUser select l;
+
+                foreach (var item in lengthOfString)
+                {
+                    Console.WriteLine(item);
+                }
+            }
+
+            //Создайте LinkedList и два элемента, вставьте второй элемент после каждого вхождения первого элемента в списке.
+            //Итак, если список равен [2,4,3,2,8,2,5,1,2], а элементы равны 2 и 10, результатом будет [2,10,4,3,2,10,8,2,10,5,1,2,10]
+
+            static void LinkedListAddTen()
+            {
+                List<string> ints = new List<string>() { "2", "4", "3", "2", "8", "2", "5", "1", "2" };
+
+                var linkedInts = new LinkedList<string>(ints);
+                var twoNode = linkedInts.First;
+                string tenNode = "10";
+
+                while (twoNode != null)
+                {
+                    if (twoNode.Value == "2")
+                    {
+                        linkedInts.AddAfter(twoNode, tenNode);
+                    }
+
+                    Console.WriteLine(twoNode.Value);
+                    twoNode = twoNode.Next;
+                }
+            }
+
+            //Объединить два связанных списка чисел, включив в окончательный список только те числа, которые встречаются в обоих списка.
+            //Итак, если списки [1,3,4,7,12] и [1,5,7,9], результат будет [1,7].
+
+            static void UniteTwoLinkedLists()
+            {
+                List<string> intsOne = new List<string>() { "1", "3", "4", "7", "12" };
+                List<string> intsTwo = new List<string>() { "1", "5", "7", "9" };
+
+                var linkedIntsOne = new LinkedList<string>(intsOne);
+                var linkedIntsTwo = new LinkedList<string>(intsTwo);
+
+                var nodeOne = linkedIntsOne.First;
+                var nodeTwo = linkedIntsTwo.First;
+
+                var smth = from one in linkedIntsOne
+                           from two in linkedIntsTwo
+                           where one.Equals(two)
+                           select two;
+
+                foreach (var item in smth)
+                {
+                    Console.WriteLine(item);
+                }
+            }
+
+            //Создайте Queue. Добавьте занчения в очередь с при помощи ввода с консоли.
+            //Реализуйте операцию: GetMaxValue, которая возвращает максимальное значение, хранящееся в данный момент в очереди, не удаляя его.
+            //Каждый раз, когда текущий максимум удаляется из очереди, операция GetMaxValue очереди должна отражать новый самый большой сохраненный элемент.
+
+            var chisla = new List<int>() { 99, 15, 3 };
```

Comment:
I guess you need to get all values for queue from console

Reply:
Fixed